src/core/mac/mac_frame.cpp

```diff
@@ -1575,16 +1750,21 @@ Frame::InfoString Frame::ToInfoString(void) const
     InfoString string;
     uint8_t    commandId, type;
     Address    src, dst;
+    uint32_t   frameCounter;
+
+    string.Append("len:%d", mLength);
 
     if (IsSequencePresent())
     {
-        string.Append("len:%d, seqnum:%d, type:", mLength, GetSequence());
+        string.Append(", seqnum:%d", GetSequence());
     }
-    else
+    else if (GetFrameCounter(frameCounter) == kErrorNone)
     {
-        string.Append("len:%d, type:", mLength);
+        string.Append(", fc:%lu", ToUlong(frameCounter));
```

Comment:
Should we add `fc` later in the `string` since it is applicable when `sec:` is enabled?